docker/s6/services/lazyraster.svc/run

```diff
@@ -1,5 +1,20 @@
 #!/bin/sh
 
+if [ -n "${MESOS_HOSTNAME}" ]; then
```

Comment:
Can we do all of this in the service instead? Seems kind of janky to have to do it in shell script when the service is designed to be a 12 factor app.